hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -761,3 +769,9 @@ void otbn_take_loop_warps(OtbnModel *model, OtbnMemUtil *memutil) {
   assert(model && memutil);
   model->take_loop_warps(*memutil);
 }
+
+void otbn_model_set_keymgr_value(OtbnModel *model, svLogicVecVal *key0,
+                                 svLogicVecVal *key1, bool valid) {
+  assert(model && key0 && key1 && valid);
```

Comment:
This is partly my fault from when we were trying to debug the segfault, but `valid` can be zero! So it shouldn't appear in the assertion here.